generation/maisi/maisi_train_vae_tutorial.ipynb

```diff
@@ -30,20 +30,9 @@
    "execution_count": 1,
```

Comment:
<p>please help remove this line: To run this tutorial, please install&nbsp;<code>xformers</code>&nbsp;by following the&nbsp;<a href="https://github.com/facebookresearch/xformers#installing-xformers" target="_blank">official guide</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done. Thanks